src/finalisers/iife.ts

```diff
@@ -80,7 +80,7 @@ export default function iife (
 
 	if (isNamespaced) {
 		wrapperIntro =
-			setupNamespace(name, 'this', false, options.globals) + wrapperIntro;
+			setupNamespace(name, typeof self !== 'undefined' ?  self : typeof global !== 'undefined' ? 'global' : 'this', false, options.globals) + wrapperIntro;
```

Comment:
It looks like this should all be a template string here?

Reply:
Pushed a new commit. Do I have to update the unit tests too?